DESCRIPTION

```diff
@@ -39,6 +40,8 @@ Suggests:
     tibble,
     tools,
     withr
+Remotes:
+    r-lib/testthat
```

Comment:
Why is this necessary?

Reply:
Because of https://github.com/r-lib/testthat/issues/1880